src/sidebar/services/annotations-exporter.tsx

```diff
@@ -152,6 +153,90 @@ export class AnnotationsExporter {
     return `${headers}\n${annotationsContent}`;
   }
 
+  buildHTMLExportContent(
+    annotations: APIAnnotationData[],
+    {
+      groupName = '',
+      displayNamesEnabled = false,
+      defaultAuthority = '',
+      /* istanbul ignore next - test seam */
+      now = new Date(),
+    }: ExportOptions = {},
+  ): string {
+    const { uri, title, uniqueUsers, replies, extractUsername } =
+      this._exportCommon(annotations, {
+        displayNamesEnabled,
+        defaultAuthority,
+      });
+
+    return renderToString(
+      <html lang="en">
+        <head>
+          <title>Annotations export - Hypothesis</title>
+          <meta charSet="UTF-8" />
+        </head>
+        <body>
+          <header>
```

Comment:
The `<header>` element is normally used for a navigation banner on a website. I think this is probably better done as a section, so you'd have something like:

```
<section>
  <h1>Summary</h1>
  ...
</section>
<section>
  <h1>Annotations</h1>
  <article>
    <h2>Annotation 1</h2>
    ...
  </article>
</section>
```